src/test_l1_dsHdmiIn.c

```diff
@@ -1717,16 +1763,16 @@ void test_l1_dsHdmiIn_positive_dsGetEDIDBytesInfo_sink(void) {
  * |05|Call dsGetEDIDBytesInfo() with invalid value |dsHDMI_IN_PORT_0, unsigned char*, NULL| dsERR_INVALID_PARAM | Should Pass |
  * |06|Call dsHdmiInTerm() to ensure deinitialization | | dsERR_NONE | Clean up after test |
  * |07|Call dsGetEDIDBytesInfo() after termination the HDMI input sub-system |dsHDMI_IN_PORT_0, unsigned char*, int*| dsERR_NOT_INITIALIZED | Should Pass |
- * 
+ *
  * @note Testing for the `dsERR_OPERATION_NOT_SUPPORTED` and `dsERR_OPERATION_FAILED` might be challenging since it requires a specific scenario where the attempted operation is not supported.
- * 
+ *
  */
 void test_l1_dsHdmiIn_negative_dsGetEDIDBytesInfo_sink(void) {
+
     gTestID = 34;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
-
     unsigned char* edidBytes = (unsigned char*) malloc(255);
-    int edidSize;
+    int edidSize = 0;
 
     // Step 1: Call dsGetEDIDBytesInfo() without initializing the HDMI input sub-system
```

Comment:
// Check if memory allocation was successful
    if (edidBytes != NULL) {
    // Add the step 1 to step 7
    }
    else {
        // Handle memory allocation failure
        UT_LOG("\n Memory allocation failed in %s\n", __FUNCTION__);
    }